### PR TITLE
[12.0][ADD] helpdesk_mgmt_ticket_tree_decoration

### PR DESCRIPTION
**Pull request by lmiguens-solvos**
Adds decoration on tickets tree view.
Put decoration according to ticket stage: red for unattended tickets, green for closed tickets.

### CODE REVIEW

File: helpdesk_mgmt_ticket_tree_decoration/views/helpdesk_ticket_views.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        <field name="inherit_id" ref="helpdesk_mgmt.ticket_view_tree"/>
+        <field name="arch" type="xml">
+            <xpath expr="." position="attributes">
+                <attribute name="decoration-danger">unattended</attribute>
+                <attribute name='decoration-success'>closed</attribute>
+            </xpath>
+            <field name="stage_id" position="after">
+                <field name="unattended" invisible="True"/>
+                <field name="closed" invisible="True"/>
+            </field>
```
Comment (dalonsod) [inline helpdesk_mgmt_ticket_tree_decoration/views/helpdesk_ticket_views.xml]: Suggested improvement (archived tickets will be shown grayed):
```suggestion
            <xpath expr="." position="attributes">
                <attribute name="decoration-muted">not active</attribute>
                <attribute name="decoration-danger">active and unattended</attribute>
                <attribute name='decoration-success'>active and closed</attribute>
            </xpath>
            <field name="stage_id" position="after">
                <field name="active" invisible="True"/>
                <field name="unattended" invisible="True"/>
                <field name="closed" invisible="True"/>
            </field>
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comment. And `__init__.py` file is empty, maybe could be removed

File: helpdesk_mgmt_ticket_tree_decoration/views/helpdesk_ticket_views.xml (hunk 1 above)
Comment (lmiguens-solvos) [reply helpdesk_mgmt_ticket_tree_decoration/views/helpdesk_ticket_views.xml]: Done! 
Although as active and non-active are never mixed, when the non-active ones are shown they will all be "muted"...

**Review by dalonsod — APPROVED** [review]
LGTM
I'll squash commits and merge it :grinning:

**Pull request merged**